packages/core/locales/pt/zod.json

```diff
@@ -13,6 +13,7 @@
     "custom": "Entrada inválida",
     "invalid_intersection_types": "Valores de interseção não poderam ser mesclados",
     "not_multiple_of": "O número deverá ser múltiplo de {{multipleOf}}",
+    "not_finite": "Number must be finite",
```

Comment:
@montedonioluiz Please tell me the translation of this message.

Reply:
Heya @aiji42!

Should be: `Número não pode ser infinito`

Reply:
@montedonioluiz Thank you!